CHANGELOG.md

```diff
@@ -3,6 +3,53 @@
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[3.10](https://github.com/nf-core/rnaseq/releases/tag/3.10)] - 2022-12-21
+
+### Enhancements & fixes
+
+- Bump minimum Nextflow version from `21.10.3` -> `22.10.1`
+- Updated pipeline template to [nf-core/tools 2.7.2](https://github.com/nf-core/tools/releases/tag/2.7.2)
+- [[#729](https://github.com/nf-core/rnaseq/issues/729)] - Add 'auto' option to samplesheet to automatically detect strandedness for samples
+- [[#889](https://github.com/nf-core/rnaseq/issues/889)] - Document valid options for `--genome` parameter
+- [[#891](https://github.com/nf-core/rnaseq/issues/891)] - Skip MarkDuplicates when UMIs are used
+- [[#896](https://github.com/nf-core/rnaseq/issues/896)] - Remove `copyTo` call for iGenomes README
+- [[#897](https://github.com/nf-core/rnaseq/issues/897)] - Use `--skip_preseq` by default
+- [[#898](https://github.com/nf-core/rnaseq/issues/898)] - Documentation on salmon decoy-aware index creation, gcbias and seqbias
+- [[#900](https://github.com/nf-core/rnaseq/issues/900)] - Add `--recursive` option to `fastq_dir_to_samplesheet.py` script
+- [[#902](https://github.com/nf-core/rnaseq/issues/902)] - `check_samplesheet.py` script doesn't output optional columns in samplesheet
+- [[#907](https://github.com/nf-core/rnaseq/issues/907)] - Add `--extra_star_align_args` and `--extra_salmon_quant_args` parameter
+- [[#912](https://github.com/nf-core/rnaseq/issues/912)] - Add UMI deduplication before quantification in tube map
+
+### Parameters
+
+| Old parameter    | New parameter               |
+| ---------------- | --------------------------- |
+| `--enable_conda` |                             |
+|                  | `--extra_star_align_args`   |
+|                  | `--extra_salmon_quant_args` |
+
+> **NB:** Parameter has been **updated** if both old and new parameter information is present.
+> **NB:** Parameter has been **added** if just the new parameter information is present.
+> **NB:** Parameter has been **removed** if new parameter information isn't present.
+
+### Software dependencies
+
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
+
+| Dependency                          | Old version | New version |
+| ----------------------------------- | ----------- | ----------- |
+| `bbmap`                             | 38.93       | 39.01       |
+| `bioconductor-dupradar`             | 1.18.0      | 1.28.0      |
+| `bioconductor-summarizedexperiment` | 1.20.0      | 1.24.0      |
+| `bioconductor-tximeta`              | 1.8.0       | 1.12.0      |
+| `fq`                                | 0.9.1       |             |
```

Comment:
Has it been added? Then it should be in the 'new' column

Reply:
👀